scripts/flying-tokens.js

```diff
@@ -69,23 +114,24 @@ export async function fly(token, elevation) {
         await flyZoom(token, elevation)
         await flyingFX(token, elevation)
         if (notificationOutput)
-            ui.notifications.info(token.actor.name + ' is flying at <b>' + elevation + ' feet</b> high.')
+            ui.notifications.info(token.name + ' is flying at <b>' + elevation + ' feet</b> high.')
         if (chatOutput)
-            await chatMessage(`<img src='${token.actor.img}' width='32' style='border:none'> ${token.actor.name} is flying at <b>${elevation} feet</b> high.`)
+            await chatMessage(`<img src='${token.texture.src}' width='32' style='border:none'> ${token.name} is flying at <b>${elevation} feet</b> high.`)
     }
 }
 
 async function tokenScale(token, elevation) {
+    pf2eAutoScaleCheck(token, false);
     let originalScale = token.getFlag(MODULE, "scale");
-    if (elevation == 0) {
-        await token.update({ scale: originalScale })
-        return 0
-    }
-    let scale = originalScale + Math.pow(elevation, 0.28)
+    let altToken = token.getFlag(MODULE, "altToken") || token.getFlag(MODULE, "originalToken");
+    if (elevation == 0) return 0
+    let scale = originalScale + customScale * elevation
     scale = Math.round((scale + Number.EPSILON) * 100) / 100 //rounding with 2 floats
-    if (scale <= 1) scale += 1
```

Comment:
this was producing big tokens lol I don't know where my mind was when I wrote this line

Reply:
Lololol